appveyor.yml

```diff
@@ -20,12 +20,6 @@ environment:
 
   matrix:
     - TARGET_ARCH: x86
-      CONDA_PY: 27
-
-    - TARGET_ARCH: x64
-      CONDA_PY: 27
-
-    - TARGET_ARCH: x86
```

Comment:
This is not OK. It is a bug in conda-smithy that we haven't been able to fix yet. Let me try with my setup and see if I can get the correct build matrix.